src/redux/reducers/commentReducer.ts

```diff
@@ -0,0 +1,85 @@
+/* External dependencies */
+import { takeLatest } from 'redux-saga/effects'
+import { OrderedMap } from 'immutable'
+
+/* Internal dependencies */
+import Comment from 'models/Comment'
+import { CommentAttr } from 'models/Comment'
```

Comment:
```suggestion
import Comment, { CommentAttr } from 'models/Comment'
```
요거 보고 제가 샘플로 만든 TodoReducer보니까 제가 샘플을 잘못 했네요ㅎㅎ
같은 파일로부터 import하는거니 저렇게 써주는게 좋을것 같아요